docs/src/content/docs/getting-started/configuration.mdx

```diff
@@ -134,20 +165,26 @@ Here's an example `workflows` configuration:
 
 ```yaml
 workflows:
-  - tag_query: "dir:production"
+  # This workflow applies to any directory tagged with "production"
+  # (This could match directories set with tags: [production] in the dirs section)
+  - tag_query: "production"
     plan:
-      - type: init
-      - type: plan
-        extra_args: ["-var-file=production.tfvars"]
+      - type: init  # Run terraform init
+      - type: plan  # Run terraform plan
+        extra_args: ["-var-file=production.tfvars"]  # With these extra arguments
     apply:
-      - type: init
-      - type: apply
-      - type: run
+      - type: init  # Run terraform init for apply
+      - type: apply  # Run terraform apply
+      - type: run   # Run a custom command
         cmd: ['echo', 'Error running apply']
-        run_on: failure
+        run_on: failure  # Only if apply fails
 ```
 
-This workflow runs a custom plan and apply process for any changes in the `production` directory.
+This workflow runs a custom plan and apply process for any directories tagged with `production`. The tag queries like `tag_query: "production"` match the tags defined in the `dirs` section, creating a connection between your directory structure and custom workflows.
+
+:::note
+All the examples in this page are complementary parts of a single configuration. The tag query in the workflow (`tag_query: "production"`) would match directories or workspaces tagged with `production` in the `dirs` section.
+:::
 
 For more information, see the [Workflows](/configuration-reference/workflows) section of the Configuration Reference.
```

Comment:
"For more information,..." – Repeats in multiple sections; use "See [section] for details" instead.